packages/datadog-instrumentations/src/vitest.js

```diff
@@ -0,0 +1,254 @@
+const { addHook, channel, AsyncResource } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+// Needs removing specifiers check in
+// node_modules/.pnpm/import-in-the-middle@1.7.3/node_modules/import-in-the-middle/index.js
+// TODO: why?
+
+// needs creating a fake file node_modules/vitest/dist/@vitest/spy
+// TODO: why?
```

Comment:
ℹ️ these lines will be removed before merging. They still need sorting out after we fix the potential issues in iitm